components/education/FeaturedModuleCard.tsx

```
@@ -50,40 +50,40 @@ export function FeaturedModuleCard({
           </div>
-          <p className="text-sm text-gray-400 line-clamp-2">{description}</p>
+          <p className="text-sm text-gray-600 dark:text-gray-400 line-clamp-2">{description}</p>
         </CardHeader>
         <CardContent>
-          <div className="flex items-center justify-between text-sm border-t border-gray-800/50 pt-4">
+          <div className="flex items-center justify-between text-sm border-t border-gray-200 dark:border-gray-800/50 pt-4">
             <div className="flex items-center gap-4">
               <div className="flex -space-x-2">
                 {instructors.map((instructor, index) => (
                   <div
                     key={index}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)